charts/seed-monitoring/charts/grafana/dashboards/owners/coredns-node-local-dns-dashboard.json

```diff
@@ -0,0 +1,2025 @@
+{
```

Comment:
Would it make sense to not enable this dashboard if the feature gate is disabled?

Reply:
Makes sense yes, but as I said, we don't yet have the logic to do that in place for the rest of the dashboards. So optionally, I can remove this dashboard this PR and later add it once we have the logic.

Reply:
Hmm, can't you do it similar to what you did with the VPN dashboard here:
https://github.com/gardener/gardener/blob/94eb4ab4fe4f05eeed58ab9aabf4235c26e586c0/charts/seed-monitoring/charts/grafana/templates/grafana-dashboards-configmap.yaml#L18-L20

Reply:
I removed monitoring from this PR.